cairo/src/Slay/Cairo/Prim.hs

```diff
@@ -84,7 +84,9 @@ data PrimCurve g =
     { curveExtents :: Extents,
       curveCurvature :: g Curvature,
       curveColor :: g Color,
-      curveDirection :: g Direction
+      curveDirection :: g Direction,
+      curveDash :: [Natural],
+      curveDashOffset :: Natural
```

Comment:
`g Unsigned`

Reply:
Actually, should the offset be non-negative? If it's not a requirement, consider making it `Signed`